roadmaps/external-deps.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: contribute
+title: Bazel External Deps Roadmap 
+---
+<style>
+  .padbottom { padding-bottom: 10px; }
+  .etabox {
+    background: #EFEFEF;
+    color: #38761D;
+    font-size: 15px;
+    font-weight: bold;
+    display: inline;
+    padding: 6px;
+    margin-right: 10px;
+  }
+  .donestatus {
+    color: #00D000;
+    font-weight: bold;
+    padding-left: 10px;
+  }
+  .inprogressstatus {
+    color: #D0D000;
+    font-weight: bold;
+    padding-left: 10px;
+  }
+</style>
```

Comment:
To ensure consistency between roadmaps, it might be simpler to avoid custom style.

I also prefer to use GitHub issues for status updates (unless you expect everyone to update the roadmap page each time an item is implemented?)

Reply:
(also, custom style is not supported by GitHub preview: https://github.com/dslomov/bazel-website/blob/3759adbb15b5cf89fd0d8fc70833701d221cd26f/roadmaps/external-deps.md)

Reply:
+1, please do not add inline styles to page. We're already having diverging style sheets across {docs, blog, }.bazel.build because they're hosted independently.